howto/stream/exchange-oob-data.md

```diff
@@ -159,6 +158,10 @@ public class DataChannelEventReceiver extends BroadcastReceiver {
 }
 ```
 
+```{note}
+If an instance is running on Android 14 image, enabling the out-of-band v2 feature requires the Android app to be running in order to receive broadcasts. If the app is in the [cached state](https://developer.android.com/guide/components/activities/process-lifecycle), the system places [context-registered broadcasts in a queue]((https://developer.android.com/develop/background-work/background-tasks/broadcasts#android-14)),meaning the app may not receive broadcasts immediately, as it would when the app is actively running.
+```
```

Comment:
What is needed for a user to do to achieve this? It's not clear to me how I can fix this without having a deep understanding of what Android does / can be influenced

Reply:
Let me reword this a bit